### PR TITLE
Support for Dark Theme in Toast

### PR DESCRIPTION
**Pull request by pooja-ps-trimble**

### CODE REVIEW

File: Trimble.Modus.Components/Trimble.Modus.Components.csproj [hunk 1]
```diff
@@ -1,273 +1,537 @@
...
+	  </MauiXaml>    
+	  <MauiXaml Update="Styles\LightTheme.xaml">
+	    <Generator>MSBuild:Compile</Generator>
+	  </MauiXaml>
+	</ItemGroup>
+	<ItemGroup>
+	  <Folder Include="Controls\Messages\" />
+	  <Folder Include="Controls\TMAlert\" />
+	</ItemGroup>
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (rthanga1) [inline Trimble.Modus.Components/Trimble.Modus.Components.csproj:265]: @pooja-ps-trimble  This is broken, I cannot even open the project. can you check?

**Review by rthanga1 — CHANGES_REQUESTED** [review]

File: Trimble.Modus.Components/Trimble.Modus.Components.csproj (hunk 1 above)
Comment (pooja-ps-trimble) [reply Trimble.Modus.Components/Trimble.Modus.Components.csproj]: @rthanga1 I fixed it

File: Trimble.Modus.Components/Controls/Toast/TMToastContents.xaml.cs [hunk 2]
```diff
@@ -16,27 +17,98 @@ public partial class TMToastContents : PopupPage
...
     {
         InitializeComponent();
-        SetTheme(theme.ToString());
+        SetTheme(theme.ToString());
+        SetDynamicResource(StyleProperty, theme.ToString());
         PopupData(message, actionButtonText, action, isDismissable);
         BindingContext = this;
         CloseAfterDelay();
+    }
+    public Color ToastBackgroundColor
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/Toast/TMToastContents.xaml.cs:52]: Can't we have them as internal?

**Review by rthanga1 — APPROVED** [review]

File: Trimble.Modus.Components/Controls/Toast/TMToastContents.xaml.cs (hunk 2 above)
Comment (pooja-ps-trimble) [reply Trimble.Modus.Components/Controls/Toast/TMToastContents.xaml.cs:52]: As the binding is implemented in xaml, it has to be public.

File: Trimble.Modus.Components/Trimble.Modus.Components.csproj (hunk 1 above)
Comment (pooja-ps-trimble) [reply Trimble.Modus.Components/Trimble.Modus.Components.csproj]: Could you check this again?

**Review by rthanga1 — APPROVED** [review]

**Pull request merged**